openshift/datasync-showcase-amq.yml

```diff
@@ -263,10 +229,14 @@ objects:
           volumeMounts:
           - mountPath: /usr/src/app/files
             name: files-storage
+          - mountPath: /website/assets/config
+            name: webapp-config
         volumes:
         - name: files-storage
           persistentVolumeClaim:
             claimName: sync-files-storage
+        - configMap:
+          name: webapp-config
     test: false
     triggers:
```

Comment:
I think this section needs to be updated as well and remove the `imageChangeParams` part.